### DISCUSSION TITLE
Upgrading from 1.3.8 to 1.3.9 breaks API - removes ch.qos.logback.classic.util.ContextInitializer.configureByResource(java.net.URL)

### DISCUSSION THREAD

**Discussion by philipwhiuk** [qos-ch/logback#696] [General]
I was surprised that a 'patch' release would break compatibility like this. It's not really documented in the release notes

Is there a logic behind this change?

**Comment by ceki** [comment]
@philipwhiuk  The deprecated method in question was removed because it was preventing the decoupling of logback initialization from JoranConfigurator. Copying the method into your project should be a relatively easy exercise.

In any case, thank you for the comment and apologies for the inconvenience.

**Comment by gaeljw** [comment]
Example of consequences: https://github.com/playframework/playframework/issues/11907

**Comment by ceki** [comment]
@gaeljw  Thank you for this link. Looking at that thread, it looks like commit [d0bf4cfbfca ]( https://github.com/playframework/playframework/pull/11910/commits/d0bf4cfbfca4ead080887764af06290272a86e98) fixes the issue.
